### PR TITLE
Adding the following possible version of the Spring Security Web

### PR DESCRIPTION
**Pull request by graflmsg**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -123,6 +123,17 @@
...
+				<exclusion>
+					<groupId>org.springframework.security</groupId>
+					<artifactId>spring-security-web</artifactId>
+				</exclusion>
+			</exclusions>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.security</groupId>
+			<artifactId>spring-security-web</artifactId>
+			<version>5.7.5</version>
```
Comment (dblankevia) [inline pom.xml:136]: We should use the latest bugfix version 5.7.7, if we have to manually upgrade the lib.

**Review by dblankevia — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (ds-mwesener) [reply pom.xml]: As far as I could see: We are currently using spring bom: 2.7.5. Which includes spring-security-web-5.7.4.

If we need to have a higher version we have to do it manually like done here.

https://docs.spring.io/spring-boot/docs/2.7.5/reference/html/dependency-versions.html#appendix.dependency-versions

Comment (dblankevia) [reply pom.xml]: We were already on Spring Boot 2.7.7 with the Gradle build. Please upgrade.

**Comment by dblankevia** [comment]
We get the transitive dependency `org.springframework:spring-web:jar:5.3.24:compile`. We have to use 5.3.25, due to vulnerabilities in 5.3.24.

**Review by dblankevia — APPROVED** [review]

**Pull request merged**